canonical/charmhub.io
# Move collaborators to react app

## Done
Moved the publisher collaborators section into the React app

## How to QA
- Go to https://charmhub-io-1814.demos.haus/<CHARM_NAME>/collaboration
- Check that it looks and works as expected (compare to live)

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-10529
